content/en/blog/_posts/2022-XX-XX-iptables-chains.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
+title: "Kubernetes’s IPTables Chains Are Not API"
+date: 2022-XX-XX
+slug: iptables-chains
+---
+
+### or, “Sorry, You Are Not Invited to SIG Network’s `MASQUERADE` Party”
+
+**Author:** Dan Winship (Red Hat)
+
+Some Kubernetes components (such as kubelet and kube-proxy) create
+IPTables chains and rules as part of their operation. These chains
+were never intended to be part of any Kubernetes API/ABI guarantees,
+but some external components nonetheless make use of some of them (in
+particular, using `KUBE-MARK-MASQ` to mark packets as needing to be
+masqueraded).
+
+In Kubernetes 1.25 we are declaring explicitly that (with one
```

Comment:
This is a blog post with one author. I strongly recommend clarifying who “we” are here - you don't mean Red Hat, I imagine.